robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -447,3 +447,114 @@ def verify_record_picklist_values(self,field,*args,**kwargs):
                 self.selenium.click_element(option_field) # clicking again to close the picklist after verification
             else:
                 raise Exception(f'{key} does not contain specified {value}')
+
+    def verify_modal_default_value(self,**kwargs):
+        """verifies that the field contains given default value
+        where key=field name and value=default value"""
+        for key,value in kwargs.items():
+            locator=npsp_lex_locators["gift_entry"]["id"].format(key)
+            type=self.selenium.get_element_attribute(locator,"data-qa-locator")
+
+            if 'textarea' in type :
+                field=npsp_lex_locators["gift_entry"]["field_input"].format(key,"textarea")
+            else:
+                field=npsp_lex_locators["gift_entry"]["id"].format(key)
+                self.selenium.wait_until_page_contains_element(field)
+                time.sleep(1)
+                element=self.selenium.get_webelement(field)
+                default_value=element.get_attribute("value")
+                assert value == default_value, f"Expected {key} default value to be {value} but found {default_value}"
+
+    def fill_modal_form(self,**kwargs):
+        """Fill the gift entry modal form fields with specified values.
+        Key is field name and value is value to be entered for field """
+        for key,value in kwargs.items():
+            locator=npsp_lex_locators["gift_entry"]["modal_id"].format(key)
+            type=self.selenium.get_element_attribute(locator,"data-qa-locator")
+            field_locator=npsp_lex_locators["gift_entry"]["modal_field"].format(key,"input")
+            print(f"type is {type}")
```

Comment:
Instead of print, I'd advise to use 'self.builtin.log'.

Reply:
thanks, appreciate your feedback.